frontend/packages/kubevirt-plugin/src/components/connected-vm-console/connected-vm-console.tsx

```diff
@@ -0,0 +1,138 @@
+import * as React from 'react';
+import { K8sResourceKind, PodKind } from '@console/internal/module/k8s';
+import { PodModel, ServiceModel } from '@console/internal/models';
+import { getVMStatus } from '../../statuses/vm/vm-status';
+import { VMImportKind } from '../../types/vm-import/ovirt/vm-import';
+import { V1alpha1DataVolume } from '../../types/vm/disk/V1alpha1DataVolume';
+import {
+  DataVolumeModel,
+  VirtualMachineImportModel,
+  VirtualMachineInstanceMigrationModel,
+  VirtualMachineInstanceModel,
+  VirtualMachineModel,
+} from '../../models';
+import { VMIKind, VMKind } from '../../types/vm';
+import { Firehose, FirehoseResult } from '@console/internal/components/utils';
+import { VmConsolesWrapper } from '../vms/vm-console';
+import { getLoadedData } from '../../utils';
+import { ConsoleEmptyState } from './vm-console-empty-state';
+import { ConsoleType } from '../../constants/vm/console-type';
+
+const CloudVMConsole: React.FC<CloudVMConsoleProps> = ({
+  type,
+  vm,
+  vmis,
+  pods,
+  migrations,
+  dataVolumes,
+  vmImports,
+}) => {
+  const loadedVM = getLoadedData(vm);
+  const loadedVMIs = getLoadedData(vmis);
+  const loadedPods = getLoadedData(pods);
+  const loadedMigrations = getLoadedData(migrations);
+  const loadedDataVolumes = getLoadedData(dataVolumes);
+  const loadedImports = getLoadedData(vmImports);
+  const vmi = loadedVMIs?.[0];
+
+  const vmStatusBundle = getVMStatus({
+    vm: loadedVM,
+    vmi,
+    pods: loadedPods,
+    migrations: loadedMigrations,
+    dataVolumes: loadedDataVolumes,
+    vmImports: loadedImports,
+  });
+
+  return (
+    <VmConsolesWrapper
```

Comment:
```suggestion
    <VMConsolesWrapper
```

Reply:
Also why "consoles" (plural) ?

Reply:
Patternfly's `AccessConsoles` is plural, so I guess someone thought its in place. ( I didnt name the components)